src/controllers/user.controller.ts

```diff
@@ -29,11 +29,8 @@ export class UserController {
   public async readCurrentUser(req: Request, res: Response): Promise<Response> {
     try {
       const { _id } = req.user!;
-      const user = await userService.getCurrentUser(_id);
-      if (user) {
-        return res.send(user);
-      }
-      return res.status(StatusCodes.NOT_FOUND).send();
+
+      return res.send(await userService.getCurrentUser(_id));
```

Comment:
What is stored in `req.user`? Isn't it the authorized user? Wouldn't it be enough to do something like this?
```
return res.send(req.user);
```

Reply:
@arkus7 req.user contains only user id and information if hes admin. It's translated user's token